### PR TITLE
Footer - add wechat/fixes

### PR DESCRIPTION
**Pull request by aliceluo2048**
image
Initially I just added wechat but then I saw the new Target Merchants banner blocks the links on desktop so I set the z-index on the links container. 

Additionally:
- Refactored social media links mapping
- Links to be opened in new tab
image

### CODE REVIEW

File: src/components/Footer/Footer.tsx [hunk 1]
```diff
@@ -68,6 +81,7 @@ const Container = styled.footer`
 `;
 
 const LinksContainer = styled.div`
+  z-index: 5;
```
Comment (justintmckibben) [inline src/components/Footer/Footer.tsx:84]: why 5?

**Review by justintmckibben — APPROVED** [review]

**Review by danthaman44 — APPROVED** [review]
Nice! Thanks for helping cleanup my tech debt

**Comment by danthaman44** [comment]
Can you verify the "Download our flyers" link works too?

**Review by jacoblb64 — APPROVED** [review]
Looks good!

**Pull request merged**